### DISCUSSION TITLE
Versión de plugin Checkmate en IntelliJ

### DISCUSSION THREAD

**Discussion by lrocampo** [dds-jv/foro#408] [Consultas de herramientas]
Buenas! Estoy siguiendo los pasos de configuración del IDE del [proyecto plantilla](https://github.com/dds-utn/java-base-project). En el paso de _Instalar y Configurar Checkstyle_ dice:
"Configurarlo activando los Checks de Google y la versión de Checkstyle `== 8.35`"
Sin embargo, en mi caso la versión más vieja que me deja elegir no es esa sino la 9.0.1
![image](https://github.com/dds-jv/foro/assets/88249238/15467a14-5986-492c-99a4-26e728a19315)

Lo menciono por las dudas.

**Comment by flbulgarelli** [comment]
Genial, gracias, vamos a actualizar el instructivo.

**Reply by flbulgarelli** [reply]
PD: ya debería estar actualizado.
